src/Ledger/PParams.lagda

```diff
@@ -72,9 +87,44 @@ paramsWellFormed pp = ⌊ ¬? (0 ∈? setFromList
   ∷ collateralPercent ∷ ccTermLimit ∷ govExpiration ∷ govDeposit ∷ drepDeposit ∷ [])) ⌋
   where open PParams pp
 \end{code}
-\caption{Definitions used for protocol parameters}
-\label{fig:defs:pparams}
+\end{AgdaAlign}
+} %% End: small
+\caption{Protocol parameter declarations}
+\label{fig:protocol-parameter-declarations}
 \end{figure*}
+The \AgdaDatatype{ProtVer} type represents the protocol version used in the Cardano ledger.
+It is a pair of natural numbers, the first of which represents the major version, the second
+represents the minor version.
+
+The \AgdaRecord{PParam} type is a record containing parameters used in the Cardano ledger, which we group according
+to the general purpose that each parameter serves. This allows thresholds to be set by group.
+\begin{itemize}
+  \item \AgdaInductiveConstructor{NetworkGroup}: parameters related to the network settings;
+  \item \AgdaInductiveConstructor{EconomicGroup}: parameters related to the economic aspects of the ledger;
+  \item \AgdaInductiveConstructor{TechnicalGroup}: parameters related to technical settings;
+  \item \AgdaInductiveConstructor{GovernanceGroup}: parameters related to governance settings.
+\end{itemize}
+We do not require that each protocol parameter governance action be confined to a single group.
+In case a governance action carries updates for multiple parameters from different groups,
+the maximum threshold of all the groups involved will apply to any given such governance action.
+
+The \emph{Network}, \emph{Economic}, and \emph{Technical} parameter groups contain protocol parameters that
+were already introduced during the Shelley, Alonzo and Babbage eras.
+The new protocol parameters introduced in the Conway era (CIP-1694) belong to the \emph{Governance} group.
+These new pramaters are declared in Figure~\ref{fig:protocol-parameter-declarations} and denote the following concepts.
+\begin{itemize}
+  \item \AgdaField{drepThresholds}: governance voting thresholds which must be equal to one of the following
```

Comment:
I think this sentence is little weird. Maybe 'governance thresholds for DReps'?

Reply:
Okay, I'll change it to 'governance thresholds for DReps.'  (Btw, most of the prose is just copied from the CIP. 😄 )